app/views/donations/new.html.erb

```diff
@@ -1,15 +1,14 @@
 <% provide(:title, "New donation: ") %>
-<div class="ali">
+<div class="">
   <%= link_to 'Back', donations_path, class: 'underline' %>
   <h2 class="my-4">New Donation</h2>
   <%= form_with model: @donation , data: { turbo: false }, class: 'flex flex-col' do |form| %>
```

Comment:
Did you disable turbo for this?

Reply:
Native flash notice was not showing on this view when the form was invalid, this was because it needed to refresh the DOM and for some reason turbo was interfering with this